### PR TITLE
Update documentation with external database configuration of 1.1.2 release

### PR DESCRIPTION
**Pull request by qboileau**

### CODE REVIEW

File: doc/Configuration.md
Comment (rokh-conduktor) [inline doc/Configuration.md:400]:
```suggestion
For quickstart purpose platform run with an internal embedded database (default).   
```

**Comment by JohnPreston** [comment]
@qboileau would you like me to guinea pig try this "before you merge"?

File: doc/Configuration.md
Comment (rokh-conduktor) [inline doc/Configuration.md:402]:
```suggestion
For production environmnents platform support (from version [**1.1.2**](../README.md#112-20-10-2022)) external database configuration. 
```

**Comment by qboileau** [comment]
Hi @JohnPreston sure, yes please do if you have time. 
Release `1.1.2` was just published :) https://hub.docker.com/repository/docker/conduktor/conduktor-platform

**Comment by JohnPreston** [comment]
Also something that I mentioned to @sderosiaux is that it'd be very nice to be able to have conduktor "build" the SQL connection string out of env vars such as
* `DB_HOST`
* `DB_USER`
* `DB_PASSWORD`
* `DB_NAME`
* `DB_PORT`

Food for thought.

**Comment by qboileau** [comment]
Yes it's on our backlog and will be done along a configuration refactoring.

**Comment by JohnPreston** [comment]
I got delayed to test, I might test tomorrow. Obviously don't wait for my feedback to merge once the work for this PR is "done"

**Review by rokh-conduktor — APPROVED** [review]

**Pull request merged**
